OBJ_TO_STL/src/OBJToSTL_Reader.cpp

```diff
@@ -0,0 +1,87 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <vector>
+#include <map>
+#include "../headers/OBJToSTL_Reader.h"
+#include "../headers/Triangles.h"
+
+// Constructor
+
+OBJToSTL_Reader::OBJToSTL_Reader() {}
+// Destructor
+
+OBJToSTL_Reader::~OBJToSTL_Reader() {}
+
+// Reads stl File.
+
+Triangulations OBJToSTL_Reader::objReader(const std::string &fileNamePath)
```

Comment:
Try to divide function into sub functions. Use modular approach. Do not exceed the block of function not more than 20 lines